langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -75,6 +80,14 @@ type XmlType xml;
 # Returns an iterator over the xml items of an xml sequence.
 #
 # # Each item is represented by an xml singleton.
+# 
+# ```ballerina
+# xml books = xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`;
+# object {
+#   public isolated function next() returns record {|xml value;|}?;
+# } iterator = books.iterator();
```

Comment:
```suggestion
# object {
#   public isolated function next() returns record {|xml value;|}?;
# } iterator = xml `<book><title>Ballerina</title></book><book><title>Java</title></book>`.iterator();
```

Why weren't these updated?

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/8bceaabd7a1a2202d2929fee34d1e15e2f2b9b18